lib/jekyll/page.rb

```diff
@@ -184,4 +184,14 @@ def write?
       true
     end
   end
+
+  class PageWithoutAFile < Page
+    # ---------------------------------------------------------------
```

Comment:
This documentation here isn't enough for justifying and explaining the existence of this class. Could we get some proper class level documentation?